models/articles.model.js

```diff
@@ -87,3 +87,22 @@ exports.updateVotesByArticleId = (article_id, inc_votes) => {
         });
     });
 };
+
+exports.removeCommentByCommentId = (comment_id) => {
+  return db
+    .query(
+      `SELECT * FROM comments
+   WHERE comment_id = $1
+`,
+      [comment_id]
+    )
+    .then((data) => {
+      if (!data.rows.length) {
+        return Promise.reject({ status: 404, msg: "comment does not exist" });
+      }
+
+      return db.query("DELETE FROM comments WHERE comment_id = $1", [
+        comment_id,
+      ]);
+    });
+};
```

Comment:
Nice chaining of your models to assert that the comment exists before you try to delete it